crm_rma_prodlot_invoice/models/stock_picking.py

```diff
@@ -27,28 +27,27 @@ class StockPicking(models.Model):
     _inherit = 'stock.picking'
 
     @api.multi
-    def action_invoice_create(self, journal_id,
-                              group=False, type='out_invoice'):
-        invoices = super(StockPicking, self).\
-            action_invoice_create(journal_id=journal_id,
-                                  group=group,
-                                  type=type)
-        if type == 'out_invoice':
-            prodlot_obj = self.env['stock.production.lot']
-            for picking in self:
-                for move in picking.move_lines:
-                    if move and move.quant_ids:
-                        lot = move.quant_ids[0].lot_id
-                        if lot.invoice_line_id:
-                            continue
-                        for inv_id in invoices:
-                            for inv_line in self.env['account.invoice'].\
-                                    browse(inv_id).invoice_line:
-                                lots = prodlot_obj.search([('invoice_line_id',
-                                                            '=',
-                                                            inv_line.id)])
-                                if inv_line.product_id.id == \
-                                        lot.product_id.id and \
-                                        len(lots) < inv_line.quantity:
-                                    lot.write({'invoice_line_id': inv_line.id})
-        return invoices
+    def _get_field_to_change_in_serial_record(self, invoice_type):
+        """ The serial/lot number can has one or two fields related
```

Comment:
I seriously ❤️  your doc blocks. Can we push this down though and give a one-line description of the method as well? The first line of a docblock should give a quick breakdown of the method, with subsequent lines getting into the nitty gritty if you so choose. Maybe something like:
```
It returns the name of the field to change in the lot record
```